src/item.cpp

```diff
@@ -2377,6 +2381,12 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     }
 }
 
+std::string item::display_money(unsigned int quantity, unsigned long charge) const {
+    std::string name = tname(quantity);
+    std::string amt = string_format(" ($%.2f)", (double)charge / 100);
```

Comment:
Same here. And if your format string is "%s%s" (and not even translated), you can just use string concatenation directly, no point in calling `string_format`.

Reply:
tname returns a translated string already